### ISSUE TITLE
NullPointerException when running any pdm task

### ISSUE THREAD

**Issue by JohnnyJayJay** [knightzmc/pdm#7]
When running `gradlew pdm` or `gradlew pdmGenDependencies`, the build fails as follows:
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':pdm'.
> java.lang.NullPointerException (no error message)
```
[**Gradle scan result**](https://scans.gradle.com/s/uq4i2rbvo7rc2)

The entire project is available at https://github.wlosp.org/presents .

It seems like the failure is connected to the `master-SNAPSHOT` version of one of the dependencies.

**Comment by JohnnyJayJay** [comment]
Can confirm this does not happen when giving it a proper tag version such as `0.1.0-alpha` instead.

**Comment by knightzmc** [comment]
This should be fixed in the latest version, but until Gradle gets its shit together, that's not going to be accessible... :(

**Comment by knightzmc** [comment]
https://github.com/gradle/plugin-portal-requests/issues/40

We're waiting on this for at least 2 issues (this and #1 )

**Comment by knightzmc** [comment]
This issue should be fixed in version `0.0.10`
